### ISSUE TITLE
Doesn't work on Android 14

### ISSUE THREAD

**Issue by dariusz-goc** [Henry-ZHR/KillDomainVerification#5]
Needs updating for Android 14.

Module enabled, framework selected in the scope. The system exhibits the same behavior as before, verified apps have verified links and no other apps can add support for that links in their opening defaults settings.

**Comment by Henry-ZHR** [comment]
Will test it myself when I can get a working Android 14 emulator

---

To make sure that you didn't do something wrong

1. Did it work on Android 13?
2. What's the log output on Android 14? (`adb logcat -s KillDomainVerification`)

**Comment by dariusz-goc** [comment]
1. Yes, worked fine on A13
2. How can I get that without a computer?

**Comment by Henry-ZHR** [comment]
Maybe running logcat from Termux?

**Comment by Henry-ZHR** [comment]
Works on my machine

![screenshot](https://github.com/Henry-ZHR/KillDomainVerification/assets/51886614/1d013d06-3f81-41e2-be99-a86d6b7515d3)

**Comment by dariusz-goc** [comment]
Can you show Play Store defaults settings?
![Screenshot_20240320-133412_Ustawienia](https://github.com/Henry-ZHR/KillDomainVerification/assets/12624870/af1c0a99-9fef-41a3-be23-22426df03e28)

**Comment by Henry-ZHR** [comment]
That's normal. You may not see any changes in setting whether the module works or not.

When you open google play links, does the system ask you which app to use instead of using Google Play without ask? If it does, then the module works.

**Comment by dariusz-goc** [comment]
No, it doesn't ask me, it just opens play store. On A13 there was a change in settings, I could uncheck individual verified urls to be supported as default (similarly to how unverified links are handled). Now they are all enabled without any option to disable.

**Comment by Henry-ZHR** [comment]
What about other links?

I can't install magisk and lsposed on avd with google play installed. I guess there may be something like backdoor for google play.

And, please provide log if possible.

**Comment by Henry-ZHR** [comment]
Seems that the related source code doesn't change. It should have the same behavior on A14 and A13. That's weird.

**Comment by dariusz-goc** [comment]
I'm traveling at the moment. I will make logs when I get back 👍
